dss/operations/secrets.py

```diff
@@ -0,0 +1,103 @@
+"""
+Get/set secret variable values from the AWS Secrets Manager
+"""
+import os
+import sys
+import click
+import boto3
+import select
+import typing
+import argparse
+import json
+import logging
+import subprocess
+
+from dss.operations import dispatch
+
+
+logger = logging.getLogger(__name__)
+
+
+events = dispatch.target("secrets",
+                         help=__doc__)
+
+
+@events.action("list-secrets")
+def list_secrets(argv: typing.List[str], args: argparse.Namespace):
+    """Print a list of all secrets"""
+    pass
```

Comment:
What does this function do? does it attempt to all the secrets under the `{secret_store}/{stage}` ?

Reply:
Forgot to fill in this placeholder!